tapir/coop/templates/coop/email/membership_confirmation.investing.body.html

```diff
@@ -2,48 +2,35 @@
 {% load i18n %}
 {% load utils %}
 {% block body %}
-    (English below)
-    <p>Liebe*r {% get_display_name_short share_owner %},</p>
+    {% blocktranslate with get_display_name_short share_owner as share_owner coop_full_name=%}
```

Comment:
```suggestion
    {% get_display_name_short share_owner as display_name %}
    {% blocktranslate with display_name=display_name %}
```

This is invalid, here is a suggestion. All the `{{ share_owner }}` inside the blocktranslate would then have to be replaced with `{{ display_name }}`